integration_tests/helpers/ffi/wallet.js

```diff
@@ -321,6 +337,22 @@ class Wallet {
     return result;
   }
 
+  getBalance() {
```

Comment:
I think this method would be more aptly named `getBalanceFromCallback()` to minimize confusion